pandas/tests/series/test_logical_ops.py

```diff
@@ -530,3 +530,19 @@ def test_int_dtype_different_index_not_bool(self):
 
         result = ser1 ^ ser2
         tm.assert_series_equal(result, expected)
+
+    def test_pyarrow_numpy_string_invalid(self):
```

Comment:
I would expect we already have tests for this behaviour, but then only in cases where it's not yet parametrized to use the string dtype?

Reply:
yes